packages/e2e-tests/specs/editor/various/toolbar-roving-tabindex.test.js

```diff
@@ -20,18 +20,16 @@ async function expectLabelToHaveFocus( label ) {
 	).toBe( label );
 }
 
-async function testBlockToolbarKeyboardNavigation(
-	currentBlockLabel,
-	currentBlockTitle
-) {
+async function testBlockToolbarKeyboardNavigation( currentBlockTitle ) {
 	await focusBlockToolbar();
 	await expectLabelToHaveFocus( currentBlockTitle );
 	await page.keyboard.press( 'ArrowRight' );
 	await expectLabelToHaveFocus( 'Move up' );
 	await page.keyboard.press( 'Tab' );
-	await expectLabelToHaveFocus( currentBlockLabel );
```

Comment:
@tellthemachines In my testing, the block wrapper can still gain focus if you use Up Arrow from an element. Not sure why it doesn't work in some tests.

Reply:
Why do we have to remove this check? Can we still test that moving focus from the toolbar to the content is still possible?

Reply:
@ellatrix I changed this because landing on the first element inside the wrapper is no longer the aria-label of the block wrapper so the test failed. I tried to still ensure moving focus to toolbar and back worked but if you press Tab from toolbar, wrapper is skipped.

Reply:
@ellatrix Rewrote the test to be less confusing. Hopefully better structure and comment helps. This test is lacking on comments as it is. Something to improve in the future.

Reply:
Another update. I could never make the tests pass. Is it fine to try to improve the tests in a follow-up PR? Essentially I need to implement some type of function for content editable blocks vs. non-content editable blocks. Content editable blocks can have the aria-label parsed directly but non-content editable blocks cannot since Up Arrow is now required to focus the block wrapper. In my opinion, these tests are still good since it checks to see if the toolbar is still accessible via Shift+Tab and Arrow keys.

Reply:
Oh, I think my E2E tests were fine but trunk is currently failing. 👎 I wish I would have checked that sooner. I wondered why they were passing locally... 😕